integration-test/int_test.bats

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+@test "test disk threshold is 20" {
+    [[ "20" = $(sudo iofog-agent info | grep "Available Disk Threshold" | awk '{print $5}') ]]
```

Comment:
This is hard coding a magic number  into the test. What happens when we decide to change the dt at some point in the future (for a valid reason). This test will fail. Better would be to test whether the returned value from Agent matches the config that we are pumping into Agent.

Reply:
Agreed. We can set these as pipeline variables

Reply:
Have updated the hardcoded number. Instead now we are reading from configuration file in the source code.

Reply:
Great!